sdk/search/azure-search-documents/azure/search/documents/_generated/models/_models_py3.py

```diff
@@ -2330,8 +2332,8 @@ class VectorQuery(_serialization.Model):
     :ivar kind: The kind of vector query being performed. Required. Known values are: "vector",
      "text", "imageUrl", and "imageBinary".
     :vartype kind: str or ~azure.search.documents.models.VectorQueryKind
-    :ivar k_nearest_neighbors: Number of nearest neighbors to return as top hits.
-    :vartype k_nearest_neighbors: int
```

Comment:
Looks like `VectorQuery`, `VectorizableImageBinaryQuery`, `VectorizableImageUrlQuery`, `VectorizableTextQuery`, and `VectorizedQuery` all have public API changes for `k_nearest_neighbors` being `renamed` to `k`. If this is intended, we should call this out in the change log.

Reply:
Good catch. They are not intentional. I have reverted the changes.